### PR TITLE
deal with the missing first/last name on agol users

### PR DESCRIPTION
**Pull request by tbock**
resolves issue #178

### CODE REVIEW

File: AGOLAccountRequestor/agol_auth.py [hunk 1]
```diff
@@ -5,6 +5,27 @@
...
+    last_name = response.get('lastName', '')
+    full_name = response.get('fullName', '')
+    if full_name != '' and first_name == '' and last_name == '':
+        full_name = full_name.split(' ')
+        first_name = full_name[0]
+        last_name = ' '.join(full_name[1:])
+
+    return {
+        'username': response.get('username'),
+        'email': response.get('email'),
```
Comment (torrin47) [inline AGOLAccountRequestor/agol_auth.py:22]: Is there any value in continuing to return full_name as part of this object, or is it completely pointless if we consistently return first and last names?  
Also... for those cases where we see lastname, firstname, is that just full_name, and so should we also check for a comma as the last character of full_name[0], and if so, we drop that character and swap the order?

**Review by torrin47 — DISMISSED** [review]
Cool kids review PRs.  Cooler kids make helpful suggestions?

File: AGOLAccountRequestor/agol_auth.py (hunk 1 above)
Comment (tbock) [reply AGOLAccountRequestor/agol_auth.py:22]: Full name is a calculated value on the internals of django based on first and last names so I'm not clear that it was ever doing anything. 
Yes if the comma format is

**Review by torrin47 — APPROVED** [review]
🥇

**Pull request merged**